osgtest/tests/test_54_gratia.py

```diff
@@ -411,10 +411,9 @@ def test_17_copy_condor_logs(self):
     #This test executes condor_meter
     def test_18_execute_condor_meter(self):
         core.state['gratia.condor-meter-running'] = False
-        core.skip_ok_unless_installed('gratia-probe-condor', 'gratia-service', 'globus-gram-job-manager-condor',
-                                      'globus-gram-client-tools', 'globus-proxy-utils')
+        core.skip_ok_unless_installed('gratia-probe-condor', 'gratia-service', 'osg-ce')
```

Comment:
With [SOFTWARE-2278](https://jira.opensciencegrid.org/browse/SOFTWARE-2278), doesn't this mean that this test will not run in the GRAM gatekeeper case? You should check out `core.skip_ok_unless_one_installed()` and list both htcondor-ce and the globus-gatekeeper packages.